packages/aws-arch/README.md

```diff
@@ -0,0 +1,90 @@
+# aws-arch (AWS Architecture)
+
+Library to provide metadata for AWS Services and AWS Resources.
+
+This package generates mappings between [@aws-cdk/cfnspec](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/cfnspec) and [AWS Architecture Icons](https://aws.amazon.com/architecture/icons/), and resolves resource metadata between these systems to infer a normalized definition of an `AwsService` and `AwsResource`.
+
+The primary aim of this package is to provide a consistent mechanism for other libraries to retrieve naming and assets associated with CloudFormation resources, and by extension CDK resources.
+
+## Get Started
+
+```sh
+yarn add '@aws-prototyping-skd/aws-arch'
+```
+
+```ts
+import { AwsArchitecture } from "@aws-prototyping-skd/aws-arch";
+
+const s3Bucket = AwsArchitecture.getResource("AWS::S3::Bucket");
```

Comment:
It would be awesome if these resource types where represented as an enum :) Do you know if the CDK already has this in one spot?

Reply:
The input for `getResource` is typed for all CfnResourceTypes parsed from CfnSpec.

<img width="797" alt="image" src="https://user-images.githubusercontent.com/464119/199877973-5dae4959-f07e-463d-83b6-f49ae7fbe241.png">

Additionally, this package exposes the generated internals as well, which as this as a Dictionary of Service=>Resource=>CfnResourceType
<img width="722" alt="image" src="https://user-images.githubusercontent.com/464119/199878143-7f9bca9f-0c94-414f-8f9b-f87b5f07375a.png">

```ts
import { aws_arch } from '@aws-prototyping-sdk/aws-arch'

AwsArchitecture.getResource(aws_arch.CfnSpec.ServiceResourceDictionary.S3.Bucket)
```

> Actually just noticed the static typing was enforced for this api, but was for the AwsResource.getResource api, I will fix this

Reply:
Discussed offline - resolving